src/test/java/com/distillery/intentsender/ui/MainToolWindowPresenterTest.kt

```diff
@@ -47,6 +48,18 @@ class MainToolWindowPresenterTest {
         verify(view, never()).displayParamsErrors(any())
     }
 
+    @Test
+    fun `component in view updated after it was selected`() {
+        val qualifiedNameStub = "stub"
+        val component: PsiClass = mock() {
+            on { qualifiedName } doReturn qualifiedNameStub
+        }
```

Comment:
Is there a way to use `PsiClass` of some existing class? This way you won't need to make a mock.

Reply:
I did not find any good way. I can do that using some Project object, but that would definitely complicate the code here.